source/guides/tooling/visual-testing.md

```diff
@@ -2,6 +2,8 @@
 title: Visual Testing
 ---
 
+# Functional vs visual testing
```

Comment:
So, I think the double header at the beginning of the article looks strange. We always start into a paragraph of text or have the 'What you'll learn' block. I didn't think the header was really necessary. 

![Screen Shot 2019-05-22 at 11 42 03 AM](https://user-images.githubusercontent.com/1271364/58148929-bc697300-7c86-11e9-848d-11ae0f449aee.png)